torch/optim/adamw.py

```diff
@@ -60,8 +62,11 @@ class AdamW(Optimizer):
             (default: False)
         maximize (bool, optional): maximize the params based on the objective, instead of
             minimizing (default: False)
-        foreach (bool, optional): whether foreach implementation of optimizer
-            is used (default: None)
+        foreach (bool, optional): whether foreach implementation of optimizer is used.
+            Since the foreach implementation is usually significantly faster than
```

Comment:
nit: This is good but maybe we want to add a note that we do this when the value is None and the user can force it on or off by passing True/False?